x-pack/test/apm_api_integration/tests/mobile/mobile_stats.spec.ts

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { APIReturnType } from '@kbn/apm-plugin/public/services/rest/create_call_apm_api';
+import { ENVIRONMENT_ALL } from '@kbn/apm-plugin/common/environment_filter_values';
+import { sumBy } from 'lodash';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+import { generateMobileData } from './generate_mobile_data';
+
+type MobileStats = APIReturnType<'GET /internal/apm/mobile-services/{serviceName}/stats'>;
+
+export default function ApiTest({ getService }: FtrProviderContext) {
+  const apmApiClient = getService('apmApiClient');
+  const registry = getService('registry');
+  const synthtraceEsClient = getService('synthtraceEsClient');
+
+  const start = new Date('2023-01-01T00:00:00.000Z').getTime();
+  const end = new Date('2023-01-01T00:15:00.000Z').getTime() - 1;
+
+  async function callApi({
```

Comment:
nit: more descriptive name
```suggestion
  async function getMobileStats({
```